addon/components/citation-widget/component.js

```diff
@@ -50,5 +53,25 @@ export default Ember.Component.extend({
                 .ajax(`${citationLink}${linkSuffix}/`, 'GET')
                 .then(resp => this.set(attr, resp.data.attributes.citation));
         }
-    }
+    },
+    _citationText: Ember.observer('selectedStyle', function() {
+        const citationLink = this.get('node.links.relationships.citation.links.related.href');
```

Comment:
Since this is also being set up in `didReceiveAttrs()`, would it make more sense to make a new variable `citationLink: null` up above and change the value of it in `didReceiveAttrs()` if needed?  Then you'd only need to make a call to that variable here.